universe/marathon.json.mustache

```diff
@@ -15,7 +15,7 @@
   "instances": 1,
   "env": {
     "CALICO_CALICOCTL_URL": "{{resource.assets.uris.calicoctl}}",
-    "CALICO_NODE_IMG": {{#calico.override-node-image?}}"{{calico.override-node-image}}"{{/calico.override-node-image}}{{^calico.override-node-image?}}"{{resource.assets.container.docker.calico-node}}"{{/calico.override-node-image}},
+    "CALICO_NODE_IMG": {{#debug.override-node-image}}"{{debug.override-node-image}}"{{/debug.override-node-image}}{{^debug.override-node-image}}"{{resource.assets.container.docker.calico-node}}"{{/debug.override-node-image}},
```

Comment:
I don't follow these changes I'm afraid - perhaps you could talk me through them.